### DISCUSSION TITLE
Pre-release notifications

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1446] [Q&A] [answered]
If one turns on notifications for a repo's release updates, will he get notified of releases marked as pre-releases too?

**Comment by kagikn** [comment, marked as answer]
Yes, you'll get notified of pre-releases if you turns on notifications for a repo's release updates. That is one of the reasons why I created a separate nightly release repo. Those who only need stable releases would get annoyed if I pushes nightly releases on this repo (this is not an emulator repo where you don't have to care about API compatibilities). 

It's a so shame that GitHub is not planning to add an option to not get notified of pre-releases 🙃
https://github.com/orgs/community/discussions/29192

**Reply by Michael21107** [reply]
I found that discussion too so I wanted to verify with someone trustworthy. I'm pretty sure that even tho I got release notifications on a repo, I didn't get notified on a pre-release so that confused me a bit (I could've just not set the notifications on but wanted to be sure). The conversation is just random people, not a GitHub dev in there, right?

**Reply by kagikn** [reply]
I keep getting pre-release notifications from [the Neovim repo](https://github.com/neovim/neovim) even when I have only the Releases box ticked in the notification settings. We have to use a separate repo so those who only want stable releases won't receive notifications about nightly releases yet will receive ones about stable releases. And yes, the conversation is just random people.

**Reply by Michael21107** [reply]
gotcha, thanks
